src/include/duckdb/common/extra_operator_info.hpp

```diff
@@ -34,6 +37,10 @@ class ExtraOperatorInfo {
 	optional_idx total_files;
 	//! Size of file list after applying filters
 	optional_idx filtered_files;
+	//! Whether sampling is pushed down into the table scan
+	bool is_sampling_pushed_down;
```

Comment:
We might want to support different sample types in the future - can we have this be a `unique_ptr<SampleOptions> sample_options` instead so that all sample options are supported?